api/v1alpha1/virtualmachineimage_conversion.go

```diff
@@ -153,25 +151,26 @@ func convert_v1alpha2_VirtualMachineImageStatusConditions_To_v1alpha1_VirtualMac
 	if securityCompliantCondition != nil {
 		v1a1Conditions = append(v1a1Conditions, *securityCompliantCondition)
 	}
-	if imageSyncedCondition != nil {
-		v1a1Conditions = append(v1a1Conditions, *imageSyncedCondition)
-	}
 	if imageProviderReadyCondition != nil {
 		v1a1Conditions = append(v1a1Conditions, *imageProviderReadyCondition)
 	}
+	if imageSyncedCondition != nil {
+		v1a1Conditions = append(v1a1Conditions, *imageSyncedCondition)
+	}
```

Comment:
Question: Is this change to reflect the order in which these were set in the controller?

Reply:
Yah. It's also the order that things are processed right above here.